packages/button/assets/index.css

```
@@ -0,0 +1,51 @@
+ * must guard navigation themselves.
+ */
+.vc-button--busy {
+    cursor: wait;
+    opacity: 0.85 !important;
+}
+
+.vc-button-spinner {
+    display: inline-block;
+    width: 1em;
+    height: 1em;
+    border: 2px solid currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)